raiden/tasks.py

```diff
@@ -50,6 +51,30 @@ def check_version():
             gevent.sleep(CHECK_VERSION_INTERVAL)
 
 
+def check_gas_escrow(raiden):
+    """ Check periodically for gas escrow in the account """
+    while True:
+        has_enough_balance, estimated_required_balance = gas_escrow.has_enough_gas_escrow(
+            raiden,
+            channels_to_open=0,
+        )
+        estimated_required_balance_eth = Web3.fromWei(estimated_required_balance, 'ether')
+
+        log.debug('Estimated safe balance', required_wei=estimated_required_balance)
```

Comment:
May be an overkill to log this in every run. Log it only when we are at the `not has_enough_balance` point